visallo/visallo
# Fix OntologyRepository when updating OWL files

Ontologies should not get re-ordered when their OWL file changes. Also, if get or create is called for a relationship with different domain/range they should be added.


Testing Instructions:
Update base.owl and restart Visallo. Ensure that there are no errors or warnings

CHANGELOG
Fixed: OWL files that are updated with new domain/range for relationships will update the ontology definition.
Fixed: The order of how the OWL files are loaded will not be changed when one is updated.
